internal/common/filespec.go

```diff
@@ -208,7 +213,7 @@ func (this *FileSpec) GetPath() string {
 
 // Copy the target of this FileSpec in the specified location.
 // If the FileSpec has a dest, then that will be the name of the resultant dirent within destDir.
-// If the filespec is a path, then copy a dirent.
+// If the filespec is a path, then copy a single dirent or all matching dirents if a glob pattern was used.
```

Comment:
I don't like the second part of this comment, it makes this seem like two cases,
but it is only one: "copy all matching dirents".